arviz/plots/plot_utils.py

```diff
@@ -372,7 +372,7 @@ def calculate_point_estimate(point_estimate, values, bw="default", circular=Fals
             x, density = kde(values, circular=circular, bw=bw)
             point_value = x[np.argmax(density)]
         else:
-            point_value = mode(values)[0][0]
+            point_value = mode(values).mode
```

Comment:
did it return a number or a ndarray? Do we care which one is returned?

Reply:
It depends on the scipy version.